examples/process-instrumentation/process-instrumentation-grafana-dashboard-sample.json

```diff
@@ -0,0 +1,640 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "id": 2,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "InQLY014z"
+      },
+      "description": "The amount of committed virtual memory for this process. One way to think of this is all the address space this process can read from without trigerring an access violation; this includes memory backed solely by RAM, by a swapfile/pagefile and by other mapped files on disk.",
```

Comment:
Something for discussion: would "this process" make sense in the UI context?

Reply:
removed the subject of the description which may be redundant in UI